modules/misunderstood/src/views/full/LanguageSwitch.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, MenuItem } from '@blueprintjs/core'
+import { Select } from '@blueprintjs/select'
+import countriesList from 'countries-list'
+import React from 'react'
+
+// TODO: put more meaningful overrides here
+// put `false` to prevent showing a flag for languages where
+// it cannot be logically determined
+const KNOWN_LANGUAGES = {
+  en: 'GB'
+}
```

Comment:
When using pro, you already have a dropdown with available languages. You could fetch the current language from the module property "this.props.contentLang"

Injected here: https://github.com/botpress/botpress/blob/master/src/bp/ui-studio/src/web/components/PluginInjectionSite/module.tsx#L147

Reply:
Yeah, already mentioned by Sylvain. Would save me a couple of hours if I knew it ahead of time :)